src/App.js

```diff
@@ -1,23 +1,186 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react';
+import './App.scss';
+import ship from './assets/ship.png';
+
+import image100 from './assets/image100.png';
+import image200 from './assets/image200.png';
+import image300 from './assets/image300.png';
+import image400 from './assets/image400.png';
+import image500 from './assets/image500.png';
+
 
 function App() {
+  // Creating states
```

Comment:
I love how you've organized and structured your code, it felt really easy to read and helped me understand your perspective as you created it!